tools/offwaketime.py

```diff
@@ -188,7 +188,13 @@ def signal_ignore(signal, frame):
     return 0;
 }
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(3, 19, 0)
+struct rq;
+
+int oncpu(struct pt_regs *ctx, struct rq *rq, struct task_struct *prev) {
```

Comment:
IIUC, `prev` should be renamed as `p`.
Or change the `p` below to `prev` to align with other files.

Reply:
@chenhengqi Fixed, thanks! :)